test/test_visualization.jl

```diff
@@ -691,6 +691,64 @@ end
             end
         end
     end
+
+    @trixi_testset "PlotData2D gives correct results" begin
+        equations = LinearScalarAdvectionEquation3D((0.2, -0.7, 0.5))
+        solver = DGSEM(polydeg = 3, surface_flux = flux_lax_friedrichs)
+        ode_solver = CarpenterKennedy2N54(williamson_condition = false)
+        coordinates_min = (-1.0, -1.0, -1.0)
+        coordinates_max = (+1.0, +1.0, +1.0)
+        diffusivity() = 5.0e-4
+        equations_parabolic = LaplaceDiffusion3D(diffusivity(), equations)
+        solver_parabolic = ViscousFormulationBassiRebay1()
+        boundary_conditions = boundary_condition_periodic
+        boundary_conditions_parabolic = boundary_condition_periodic
+        initial_refinement_level=3
+
+        function initial_condition_diffusive_convergence_test(x, t,
+                                                                      equation::LinearScalarAdvectionEquation3D)
+            # Store translated coordinate for easy use of exact solution
+            x_trans = x - equation.advection_velocity * t
+            nu = diffusivity()
+            c = 1.0; A = 0.5; L = 2; f = 1 / L; omega = 2 * pi * f
+            scalar = c + A * sin(omega * sum(x_trans)) * exp(-2 * nu * omega^2 * t)
+            return SVector(scalar)
+        end
+        ic = initial_condition_diffusive_convergence_test
```

Comment:
```suggestion
        initial_condition = initial_condition_diffusive_convergence_test
```

We keep it usually a little more verbose